.github/workflows/sync_project.yml

```diff
@@ -0,0 +1,33 @@
+name: Sync qube project
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Setup Python
+          uses: actions/setup-python@v1
+          with:
+            python-version: 3.8
+
+        - name: Install qube
+          run: pip install qube
+
+        - uses: actions/checkout@v2
+          with:
+              fetch-depth: 0
```

Comment:
```suggestion
              fetch-depth: 0
              token: '${{ secrets.JOHNNY_Q5_REPORTS_TOKEN}}'
```

Reply:
Disclaimer: I'm assuming here that's needed for all projects, Sven added that to the projectwizard.

Reply:
Nice catch! I am not sure maybe it needs to be specified for this repo also?

Reply:
I think so, since it seems to have to do with checkout and is not portlet-specific.